messenger/Cargo.toml

```diff
@@ -11,15 +11,19 @@ documentation = "https://docs.rs/solana-geyser-plugin-interface"
 solana-geyser-plugin-interface  = { version = "=1.10.10" }
 plerkle-serialization={path= "../plerkle_serialization"}
 redis = {version = "0.21.5", features = ["aio", "tokio-comp", "streams"], optional = true}
+pulsar={git = "https://github.com/VadimGrozinok/pulsar-rs.git", optional = true}
```

Comment:
Everything I did is update zstd crate version because there was versions conflict
Also you can find a [PR](https://github.com/streamnative/pulsar-rs/pull/218) I did to the official repo, so once it is accepted we can switch to the official repo